### ISSUE TITLE
Investigate devnet issues for migrator

### ISSUE THREAD

**Issue by MaksymilianDemitraszek** [software-mansion/protostar#792]
```
FAILED tests/integration/migrator/migrator_invoke_cheatcode_test.py::test_happy_case - protostar.commands.test.test_environment_exceptions.StarknetRevertableException: [type] TRANSACTION_FAILED
FAILED tests/integration/migrator/migrator_invoke_cheatcode_test.py::test_waiting_for_acceptance - protostar.commands.test.test_environment_exceptions.StarknetRevertableException: [type] TRANSA...
=
```

**Comment by Arcticae** [comment]
This was investigated. Will be most probably fixed by devnet 0.10 support release

**Comment by mkaput** [comment]
> This was investigated. Will be most probably fixed by devnet 0.10 support release

v0.3.0 promises to partially support Cairo 0.10... and we do already use it

**Comment by Arcticae** [comment]
> > This was investigated. Will be most probably fixed by devnet 0.10 support release
> 
> v0.3.0 promises to partially support Cairo 0.10... and we do already use it

Yes and it was tested on testnet - it works on testnet, but not on devnet. Starknet.py has this issue as well and ignores the test failures for now

**Comment by Arcticae** [comment]
This said - i'd leave #789 open until we bump devnet to a version that has this patched
